packages/firebase_app_check/firebase_app_check/android/src/main/java/io/flutter/plugins/firebase/appcheck/FlutterFirebaseAppCheckPlugin.java

```diff
@@ -77,9 +77,16 @@ private Task<Void> activate(Map<String, Object> arguments) {
     cachedThreadPool.execute(
         () -> {
           try {
+            // To test your app in debug mode, you may uncomment the following
```

Comment:
@Lyokone what do you think about uncommenting this code and having the debug provider as default? Or maybe we could pass in another value (something like `androidDebugProvider` boolean) into the activate function to use the debug provider?